packages/steps/src/BuildConfigParser.ts

```diff
@@ -180,14 +204,14 @@ export class BuildConfigParser {
 
   private createBuildStepOutputsFromDefinition(
```

Comment:
nit pick: inconsistent naming for createBuildStepOutputsFromDefinition createBuildStepInputsFromBuildStepInputsDefinition